src/main.ts

```diff
@@ -134,9 +163,12 @@ const gameLoop = (currentTime: number) => {
     // update game objects
     hummingbird && hummingbird.update(frameDelta)
     pipeController && pipeController.update(frameDelta)
+    incrementPoints()
+    showPointCounter()
   }
 
   if (!gameOver && hummingbird && pipeController && (hummingbird.collide() || pipeController.collideWith(hummingbird))) {
+    if (gamePoints > highScore) highScore = gamePoints
```

Comment:
we could just make this `highscore = Math.max(gamePoints, highScore)` it's the same in terms of performance but maybe just a little nicer to read? Also fine to not do it as there is no performance increase

Reply:
Yeah I think that's easier to read